rules.adoc

```diff
@@ -775,7 +775,7 @@ ball.
 [[regulations-inference-in-lightweight]]
 ==== Infrared interference in Lightweight
 
-In Lightweight, the robot must not emit infrared light.
+Components designed to emit IR (e.g. IR time-of-flight LiDAR sensors) are not allowed and must be covered or removed if present at all. Any device designed to emit light of any wavelength must be mounted so that light is directed perpendicular to the playing field or be covered/removed.
```

Comment:
As this is a change from last year's rules it needs to be encapsulated in {~~Old Text~>New Text~~} in order to get marked up as a change in the regular way (entry in list, red text, footnote).

Reply:
sorry - still getting bearings on ascii docs as well as git.

Reply:
No worries, that's not vanilla AsciiDoc, it's an additional markup feature specifically for Soccer rules that @mrshu implemented as a preprocessing step in the build process.